src/Helpers.py

```diff
@@ -68,6 +68,7 @@ def is_category_enabled(multiworld: MultiWorld, player: int, category_name: str)
 def resolve_yaml_option(multiworld: MultiWorld, player: int, data: dict) -> bool:
     if "yaml_option" in data:
         for option_name in data["yaml_option"]:
+            option_name = option_name.strip().replace(" ", "_")

```

Comment:
This would be a good candidate for a helper function, since it's repeated in 5 places. Also, if we ever change how we sanitize/format option names, we'll have it in one place with a function. :smiley: